src/main/java/sirius/biz/process/ExportLogsAsFileTaskExecutor.java

```diff
@@ -207,7 +218,12 @@ private void exportTable(ExportFileType exportFileType, ProcessOutput out, Proce
             List<String> columns = tableProcessOutputType.determineColumns(out);
             List<String> labels = tableProcessOutputType.determineLabels(out, columns);
             export.addRow(labels);
-            createLogsQuery(out, processContext).iterateAll(logEntry -> writeTableRow(columns, export, logEntry));
+            AtomicInteger rowCount = new AtomicInteger(0);
+            createLogsQuery(out, processContext).iterateAll(logEntry -> {
+                writeTableRow(columns, export, logEntry);
+                setExportState(processContext, rowCount.incrementAndGet(), false);
```

Comment:
or simply processContext.incCounter("$Process.rowsExported");

Reply:
I found the state "nicer" then just a count. The state can also be seen from the job list view.